stage1/style.css

```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto');
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.calculator-wrapper {
```

Comment:
I'd make this root class just `.calculator` and then all classnames used within this element should have `.calculator-` for a prefix.